doc/manuals/python/process.ipynb

```diff
@@ -0,0 +1,317 @@
+    "\n",
+    "This process serves as the only node in the pipeline, accepting and producing no output ports. A more substantive example will follow."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "class HelloWorldPython(process.KwiverProcess):\n",
+    "    def __init__(self, conf):\n",
+    "        super().__init__(self,conf)\n",
+    "        flags = kwiver.sprokit.pipeline.process.PortFlags()\n",
+    "        flags.add(self.flag_required)\n",
+    "    def _configure(self):\n",
+    "        pass\n",
+    "    def _step(self):\n",
+    "        sys.stdout.write(\"hello world\")\n",
+    "        self._base_step()"
+   ]
+  },
+  {
+   "source": [
+    "\\# Pipefile for this process\n",
+    "\n",
+    "#==================================================================\n",
+    "\n",
+    "process helloworld\n",
+    "\n",
+    "  :: hello_world_process\n",
+    "\n",
+    "#==================================================================\n",
+    "\n",
+    "config _scheduler\n",
+    "\n",
+    "  type = pythread_per_process\n",
+    "\n",
+    "\\# -- end of file --"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "Next we will detail the differentiation between inheriting from `process.KwiverProcess` and `process.PythonProcess`"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "Pure python process implementing functionality from python\n",
+    "\n",
+    "For this example we will be implementing a *very* simple 3d point class"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "class pt3d(object):\n",
+    "    def __init__(self, x = 0, y = 0, z = 0, dtype = np.float32):\n",
+    "        self.x = x\n",
+    "        self.y = y\n",
+    "        self.z = z\n",
+    "        self.dtype = dtype\n",
+    "    def as_array(self):\n",
+    "        return np.ndarray((1,3), dtype=self.dtype, buffer=[self.x,self.y,self.z])\n",
+    "    def as_mat(self):\n",
+    "        return np.matrix(self.as_array())\n",
+    "    def __str__(self):\n",
+    "        return \"(%s)\" % \",\".join(str(self.x), str(self.y), str(self.z))"
+   ]
+  },
+  {
+   "source": [
+    "Now that we have a basic data type, we need to create processes that can handle this datatype. Preexisting Kwiver processes will not support it as this data type is outside of vital. For demonstration purposes, the KwiverProcess utility mixin will leveraged to improve clarity and readability."
```

Comment:
"mixin will leveraged to improve clarity" .... needs words 'mixin will be leveraged ...'?